theatre/studio/src/panels/DetailPanel/prop-editors/simple-editors/BooleanPropEditor.tsx

```diff
@@ -0,0 +1,26 @@
+import type {PropTypeConfig_Boolean} from '@theatre/core/propTypes'
+import React, {useCallback} from 'react'
+import styled from 'styled-components'
+import BasicCheckbox from '@theatre/studio/uiComponents/form/BasicCheckbox'
+import type {ISimplePropEditorReactProps} from './ISimplePropEditorReactProps'
+
+const Input = styled(BasicCheckbox)`
+  margin-left: 6px;
+`
+
+function BooleanPropEditor({
+  propConfig,
+  editingTools,
+  value,
+}: ISimplePropEditorReactProps<PropTypeConfig_Boolean>) {
+  const onChange = useCallback(
+    (el: React.ChangeEvent<HTMLInputElement>) => {
+      editingTools.permanentlySetValue(Boolean(el.target.checked))
+    },
+    [propConfig, editingTools],
+  )
+
+  return <Input checked={value} onChange={onChange} />
+}
+
+export default BooleanPropEditor
```

Comment:
2/10 let's use `/prpoEditors/simpleEditors` for the path rather than `/prop-editors/simple-editors`

Reply:
I feel 2/10 that we should avoid confusion about actual identifiers in path names versus "category-of".

My gripe is that I would expect there to be a `propEditor` item somewhere in the code, because of the consistency with ReactComponent named files.

WDYT?

Reply:
I've moved it for now.